File: sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/FileLines9.vb
'SonarAnalyzer for .NET
'Copyright(C) 2015-2019 SonarSource SA
'mailto: contact AT sonarsource DOT com

'This program Is free software; you can redistribute it And/Or
'modify it under the terms Of the GNU Lesser General Public
'License as published by the Free Software Foundation; either
'version 3 Of the License, Or (at your option) any later version.

'This program Is distributed In the hope that it will be useful,
'but WITHOUT ANY WARRANTY; without even the implied warranty Of
'MERCHANTABILITY Or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
'Lesser General Public License For more details.

'You should have received a copy Of the GNU Lesser General Public License
'along with this program; if Not, write To the Free Software Foundation,
'Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.

Namespace Tests.Diagnostics
    Class Foo
        Sub New() ' TEST
            ' HELLO
            REM This is also a syntax for line comment
        End Sub
    End Class

End Namespace


' Some comment here

File: analyzers/tests/SonarAnalyzer.Test/TestCases/FileLines9.cs
/*
 * SonarAnalyzer for .NET
 * Copyright (C) 2014-2025 SonarSource SA
 * mailto:info AT sonarsource DOT com
 * This program is free software; you can redistribute it and/or
 * modify it under the terms of the Sonar Source-Available License Version 1, as published by SonarSource SA.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 * See the Sonar Source-Available License for more details.
 *
 * You should have received a copy of the Sonar Source-Available License
 * along with this program; if not, see https://sonarsource.com/license/ssal/
 */

namespace Tests.Diagnostics
{
    public class Foo /*
        dsad
        dsad
        sad
        sa
        dasd
        sa
        dasd
        sad
        as
        */
    {
        /// <summary>
        ///
        /// </summary>
        public Foo() { // TEST
            string multilineToken = @"

                ";
} } }


// hello
